evalml/pipelines/utils.py

```diff
@@ -637,8 +639,22 @@ def generate_pipeline_code(element):
             element.__class__.__name__,
         ),
     )
+    has_dfs = "DFS Transformer" in element.component_graph.compute_order
```

Comment:
I think we might want to reference the name attribute of the class here rather than use a string.  If we change the name of the DFS Transformer, which I think is likely, then this error will appear again and it might be unclear why.